.config/factory.js

```diff
@@ -0,0 +1,33 @@
+import { baseConfig } from './base';
+import { cjsConfig } from './commonjs';
+import { esConfig } from './es';
+import { umdConfig } from './umd';
+import { minConfig } from './minified';
+
+export function createConfig() {
+  const env = process.env.NODE_ENV;
+  const config = baseConfig;
+  const newConfigs = {
+    cjs: cjsConfig,
+    es: esConfig,
+    umd: umdConfig,
+    min: minConfig,
+  };
+  const newConfig = newConfigs[env];
+
+  for ( let key in newConfig ) {
+    if (newConfig.hasOwnProperty(key)) {
```

Comment:
Maybe it should be more like in our helper? https://github.com/handsontable/handsontable/blob/master/src/helpers/object.js#L228

Reply:
Is there a reason for how it's made in the helper?